subprojects/jacoco/src/integTest/groovy/org/gradle/testing/jacoco/plugins/JacocoAggregationIntegrationTest.groovy

```diff
@@ -324,6 +324,64 @@ class JacocoAggregationIntegrationTest extends AbstractIntegrationSpec {
         report.assertHasClassCoverage("transitive.Powerize")
     }
 
+    def 'can apply custom attributes to refine coverage results'() {
```

Comment:
Do you think a second example where there are 2 test tasks within a single project that need to be differentiated might also help?  These tests seem very clear to me, but I wonder if the use of different projects (and one of them transitively included) here might add some opacity to test suite beginners.

Reply:
I thought about it, and yes that more closely mirrors the android use-case. Then again, it also feels more like we are testing variant resolution.